x-pack/platform/plugins/private/observability_ai_assistant_management/common/ui_settings.ts

```diff
@@ -14,6 +14,33 @@ import {
   aiAssistantAnonymizationRules,
 } from '@kbn/observability-ai-assistant-plugin/common';
 
+const baseRuleSchema = schema.object({
+  id: schema.string(),
+  enabled: schema.boolean(),
+  description: schema.maybe(schema.string()),
+});
+
+const regexRuleSchema = schema.allOf([
+  baseRuleSchema,
+  schema.object({
+    type: schema.literal('regex'),
+    pattern: schema.string(),
+    entityClass: schema.string(),
```

Comment:
`entityClass` will be away for user to enable/disable specific NER entities of type of PER, ORG, LOC. We could potentially also style those classes like ML UI does. For Regex, at the moment, they're ignored, and perhaps don't need to be required.